packages/salesforcedx-vscode-core/test/vscode-integration/commands/forceProjectCreate.test.ts

```diff
@@ -124,13 +124,6 @@ describe('Force Project Create', () => {
       expect(response.type).to.equal('CONTINUE');
       expect((response as ContinueResponse<ProjectName>).data.projectName).to.equal(PROJECT_NAME);
     });
-
-    it('Should return Continue with trimmed project name if project name input has leading and or trailing spaces', async () => {
```

Comment:
Discussed in 11sies it may be good to re-write this to ensure coverage of the scenario, even if this specific test is no longer valid (won't ever get to the point of having a trimmed project name)